### PR TITLE
Add Skeleton screen to Navigator

### PR DESCRIPTION
**Pull request by marinaaisa**
Bug/issue #90172650, if applicable: 

## Summary

Add Skeleton screen to Navigator

https://user-images.githubusercontent.com/8567677/193631610-c4a0bffa-3110-4759-a7d2-7e8154548e5e.mov

## Dependencies

NA

## Testing

Steps:
1. Open the project and go to `src/components/Navigator.vue`
2. Set line 17 to `:isLoading="true"`
3. Run the project
4. Assert that you can see the skeleton screen

## Checklist

Make sure you check off the following items. If they cannot be completed, provide a reason.

- [x] Added tests
- [x] Ran `npm test`, and it succeeded
- [x] Updated documentation if necessary

### CODE REVIEW

**Comment by marinaaisa** [comment]
@swift-ci test

File: src/components/Navigator/BaseNavigatorCardItem.vue [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+  Copyright (c) 2022 Apple Inc. and the Swift project authors
+  Licensed under Apache License v2.0 with Runtime Library Exception
+
+  See https://swift.org/LICENSE.txt for license information
+  See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+-->
+
+<template>
+  <div
+    v-bind="$attrs"
```
Comment (dobromir-hristov) [inline src/components/Navigator/BaseNavigatorCardItem.vue:13]: I think if you are binding to the top it should just pass them through, no need for `v-bind="$attrs"`.

File: tests/unit/components/Navigator/NavigatorCardItem.spec.js [hunk 2]
```diff
@@ -311,8 +313,7 @@ describe('NavigatorCardItem', () => {
     });
 
     expect(wrapper.find(TopicTypeIcon).exists()).toBe(false);
-    expect(wrapper.find('.navigator-icon').classes())
-      .toEqual(expect.arrayContaining(['changed', 'changed-modified']));
+    expect(wrapper.find('.changed.changed-modified'));
```
Comment (dobromir-hristov) [inline tests/unit/components/Navigator/NavigatorCardItem.spec.js:316]: I think you missed the assertion here :)

**Review by dobromir-hristov — COMMENTED** [review]
Overall looks fine. I noticed some visual issues:

Tags are pushed down
image
image

Items overflow, dont trunkate.
image

**Comment by marinaaisa** [comment]
@swift-ci test

**Review by hqhhuang — APPROVED** [review]

**Comment by marinaaisa** [comment]
@swift-ci test

**Pull request merged**